src/a01/cli/command_definition.py

```diff
@@ -39,14 +41,19 @@ def execute(self, args: argparse.Namespace):
 
         return self.func(**kwargs)
 
+    def output(self, arg: argparse.Namespace):
+        result = self.execute(arg)
+        if isinstance(result, CommandOutput):
```

Comment:
>isinstance [](start = 11, length = 10)

IsInstance?

Reply:
Yeah is `isinstance` is a built in method. ...